common/src/main/resources/assets/cloth-config2/lang/pl_pl.json

```diff
@@ -0,0 +1,51 @@
+{
+    "text.cloth-config.save_and_done": "Zapisz i wyjdź",
+    "text.cloth-config.apply": "Zastosuj zmiany",
+    "text.cloth-config.quit_config": "Niezapisane zmiany",
+    "text.cloth-config.quit_config_sure": "Czy na pewno chcesz zakończyć edycję? Zmiany nie zostaną zapisane!",
+    "text.cloth-config.cancel_discard": "Odrzuć zmiany",
+    "text.cloth-config.quit_discard": "Wyjdź i odrzuć zmiany",
+    "text.cloth-config.config": "Konfiguracja",
+    "text.cloth-config.multi_error": "Wiele problemów!",
+    "text.cloth-config.not_editable": "Nie można edytować!",
+    "text.cloth-config.error.not_valid_number_int": "Nieprawidłowa liczba! (Integer/Całkowita)",
+    "text.cloth-config.error.not_valid_number_long": "Nieprawidłowa liczba! (Long/Długa)",
+    "text.cloth-config.error.not_valid_number_float": "Nieprawidłowa liczba! (Float/Zmiennoprzecinkowa)",
+    "text.cloth-config.error.not_valid_number_double": "Nieprawidłowa liczba! (Double/Podwójna)",
```

Comment:
Długa should also just be całkowita,
Podwójna should also just be zmiennoprzecinkowa (though it should also be a better translation, but couldn't come up with one atm)

Reply:
I think it would be best to leave it as it was before and not try to translate it at all. What do you think?

Reply:
Probably better off leaving it as that unfortunately.